src/autolabel/models/refuelV2.py

```diff
@@ -0,0 +1,232 @@
+import asyncio
+import json
+import os
+import requests
+import logging
+from typing import List, Optional, Tuple
+from time import time
+import httpx
+
+from autolabel.models import BaseModel
+from autolabel.configs import AutolabelConfig
+from autolabel.cache import BaseCache
+from autolabel.schema import LabelingError, ErrorType, RefuelLLMResult
+
+from tenacity import (
+    before_sleep_log,
+    retry,
+    stop_after_attempt,
+    wait_exponential,
+    retry_if_not_exception_type,
+)
+from langchain.schema import Generation
+
+UNRETRYABLE_ERROR_CODES = [400, 422]
+logger = logging.getLogger(__name__)
+
+
+class UnretryableError(Exception):
+    """This is an error which is unretriable from autolabel."""
+
+
+class RefuelLLMV2(BaseModel):
+    DEFAULT_TOKENIZATION_MODEL = "NousResearch/Llama-2-13b-chat-hf"
```

Comment:
can we use llama 3 instead

Reply:
Let me take care of tokenization stuff as a followup. That will be part of the big autolabel cleanup that's coming up next.